sources/packages/api/src/database/entities/student-file.model.ts

```diff
@@ -63,4 +64,29 @@ export class StudentFile extends RecordDataModel {
     referencedColumnName: ColumnNames.ID,
   })
   student: Student;
+  /**
+   * File original originated from like Application or
+   * Student uploader form. If its Temporary, then the
+   * file is uploaded but the file uploaded form is not
+   * submitted, when the form is submitted, the file
+   * origin is updated from Temporary to the respective
+   * file_origin_type;
+   */
+  @Column({
+    name: "file_origin",
+    nullable: false,

```

Comment:
Do we need to add the _enum_ properties _enum/enumName_ in the same way that we did with the other after the Nest upgrade?
@dheepak-aot FYI